scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -395,12 +395,12 @@ static struct task_ctx *lookup_task_ctx_may_fail(struct task_struct *p)
 
 static struct task_ctx *lookup_task_ctx(struct task_struct *p)
 {
-	struct task_ctx *tctx = lookup_task_ctx_may_fail(p);
+    struct task_ctx *tctx = lookup_task_ctx_may_fail(p);
 
-	if (!tctx)
-		scx_bpf_error("task_ctx lookup failed");
+    if (!tctx)
+        scx_bpf_error("task_ctx lookup failed");
 
-	return tctx;
+    return tctx;
```

Comment:
I think currently the C files are using tabs for indentation. Probably best to stay consistent.

Reply:
ahh you know I keep tripping up over that, will fix but will also see if I can add an config so editors etc. do the right thing. knowing tab not space helps, thx.